pkg/config/config_test.go

```diff
@@ -62,6 +64,25 @@ var _ = Describe("Config", func() {
 			Expect(err).To(HaveOccurred())
 		})
 	})
+	It("Assuming incorrect config file - invalid trunk config", func() {
```

Comment:
can you add more test to invalid trunk config. 
e.g. negative numbers. vlan out of range and  more.

Reply:
done